CHANGELOG.rst

```diff
@@ -1,6 +1,25 @@
 Changelog
 =========
 
+3.4.1
```

Comment:
Should this be 3.5.0 for the new feature?

Reply:
Yeah, probably. Fixing.